packages/bpk-mixins/src/mixins/_badges.scss

```diff
@@ -31,13 +31,29 @@
 ///   }
 
 @mixin bpk-badge {
-  display: inline-block;
+  display: inline-flex;
```

Comment:
I'm a little worried that this change can break layout of badges alongside other elements. Specifically, I worry that switching from `display: inline-block` to `display: inline-flex` would make the container have a similar display property as `display: inline`.

Reply:
It might be ok https://codesandbox.io/s/l2118o538l

Reply:
Yeah I did some googling before I made the change and I think it's ok. 

From what I can tell, `inline-flex` is to `flex` as `inline-block` is to `block`. So it should behave the same. https://stackoverflow.com/a/46864624/1011161

Reply:
Also your example wasn't really applicable to this scenario IMO as you put block-level elements inside the wrappers. This one makes more sense, I think: https://codesandbox.io/s/8yk138z9jj

Reply:
Updated the example to add `margin-top: 30px` to each element. `inline` doesn't respect top and bottom margins, but you can see that `inline-flex` does. So I think it's safe.